### PR TITLE
Move docs

### LINKED ISSUE #120: [DOC] How to use pre-emptible and spot instances
### Preliminary Checks

- [X] This issue is not a question, feature request, RFC, or anything other than a bug report. Please post those things in GitHub Discussions: https://github.com/nebari-dev/nebari/discussions

### Summary

[notion page](https://www.notion.so/quansightlabs/Preemptible-and-spot-instances-7fd3b26646b840a481bb2ce283ed6454)

Migrate/update old docs: https://docs.qhub.dev/en/latest/source/admin_guide/preemptible-spot-instances.html

### Steps to Resolve this Issue

- Add into `docs/FAQ`

### PR DESCRIPTION
**Pull request by ppwadhwa**
This PR moves spot instance information to the docs/FAQ.md document.  It fixes #120.

Does it need to be a how-to doc instead?

## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features not to work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [ ] Did you test the pull request locally?
- [ ] Did you add new tests?

#### Text styling

- [ ] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [ ] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [ ] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [ ] This content adheres to the Nebari style guides.

#### Non-text content

- [ ] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [ ] If there are emojis, there are not more than three in a row.
- [ ] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [ ] If the content were to be read as plain text, it still makes sense, and no information is missing.

### CODE REVIEW

**Comment by kcpevey** [comment]
@ppwadhwa Take a look at the Netlify preview to see how it looks when built. The level of the headers needs some adjustment https://deploy-preview-261--nebari-docs.netlify.app/docs/faq

**Comment by ppwadhwa** [comment]
@kcpevey How does it look now?

File: docs/docs/faq.md [hunk 1]
```diff
@@ -76,6 +76,49 @@ The `conda` config is located in the `/home/{user}/.condarc` file. You can chang
 
 You may find that the pods hosting your environment get full over time, prompting you to clear them out. To delete old builds of your environment on conda-store, click the "delete" button in the conda-store UI.
 
+## What are Preemptible and Spot instances on Nebari?
```
Comment (pavithraes) [inline docs/docs/faq.md:79]:
```suggestion
## What are preemptible and spot instances on Nebari?
```

We prefer sentence case for headings for the docs :) Ref:https://www.nebari.dev/docs/community/style-guide#capitalization

Comment (pavithraes) [reply docs/docs/faq.md:79]: Actually, I think "How to use preemptible and spot instances on Nebari?" might be a better fit here, what do you think?

**Review by pavithraes — APPROVED** [review]
@ppwadhwa Thank you for this PR! I've shared a small suggestion, but I think this is good to go after that change so I'll approve. :)

**Review by kcpevey — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**